plugins/hydra_submitit_launcher/hydra_plugins/hydra_submitit_launcher/conf/hydra/launcher/submitit.yaml

```diff
@@ -21,15 +21,17 @@ params:
       # Maximum number of retries on job timeout.
       # Change this only after you confirmed your code can handle re-submission
       # by properly resuming from the latest stored checkpoint.
-      max_num_timeout: 1
+      max_num_timeout: 0
     # local queue parameters
     local:
       gpus_per_node: 1
       tasks_per_node: 1
       timeout_min: 60
     # auto queue parameters
     auto:
-      max_num_timeout: 1
+      slurm_max_num_timeout: 0
```

Comment:
max_num_timeout was deprecated for auto executor, since it is currently slurm specific